mypyc/codegen/emitfunc.py

```diff
@@ -464,6 +464,13 @@ def visit_binary_int_op(self, op: BinaryIntOp) -> None:
         self.emit_line('%s = %s%s %s %s%s;' % (dest, lhs_cast, lhs,
                                                op.op_str[op.op], rhs_cast, rhs))
 
+    def visit_load_mem(self, op: LoadMem) -> None:
+        dest = self.reg(op)
+        src = self.reg(op.src)
+        # TODO: we shouldn't dereference to type that are pointer type so far
+        type = self.ctype(op.type)
+        self.emit_line('%s = *(%s*)%s;' % (dest, type, src))
```

Comment:
A tiny nit: in C it's more common to add a space before `*` in a pointer type (e.g., `char *` instead of `char*`).